plugins/github-pull-requests-board/src/plugin.ts

```diff
@@ -28,7 +30,9 @@ const githubPullRequestsBoardPlugin = createPlugin({
 });
 
 /** @public */
-export const EntityTeamPullRequestsCard = githubPullRequestsBoardPlugin.provide(
+export const EntityTeamPullRequestsCard: (
+  props: EntityTeamPullRequestsCardProps,
+) => JSX.Element | null = githubPullRequestsBoardPlugin.provide(
```

Comment:
I don't think we need to specify any types here or set the types. Typescript should be able to infer all the types. See
https://github.com/backstage/backstage/blob/00fed53691795a9539e8c651b45eb03280877cfe/plugins/home/src/plugin.ts#L145 and https://github.com/backstage/backstage/blob/master/plugins/home/src/homePageComponents/HeaderWorldClock/HeaderWorldClock.tsx#L106

 for example.